src/sentry/mail/actions.py

```diff
@@ -34,7 +35,7 @@ def render_label(self) -> str:
             self.data = {**self.data, "fallthroughType": FallthroughChoiceType.ACTIVE_MEMBERS.value}
         return self.label.format(**self.data)
 
-    def after(self, event, state):
+    def after(self, event, state, notification_uuid: Optional[str] = None):
```

Comment:
@jangjodi should we add `**kwargs` to all the `after` methods just in case?

Reply:
@jangjodi Yea just a bit safer if we add it I think. If we mess this up it's an incident.